api/test/pact/pact-tests/rdCommon/getCaseFlagRefData.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { expect } from 'chai';
+import * as config from 'config';
+import * as sinon from 'sinon';
+import { mockReq, mockRes } from 'sinon-express-mock';
+import { PactTestSetup } from '../settings/provider.mock';
+import { getRdCommonDataAPIOverrides } from '../utils/configOverride';
+import { requireReloaded } from '../utils/moduleUtil';
+
+const { Matchers } = require('@pact-foundation/pact');
+const { somethingLike } = Matchers;
+const pactSetUp = new PactTestSetup({ provider: 'referenceData_commonData', port: 8000 });
+
+const serviceId = 'IA';
+
+describe('RD get case flag ref data', async () => {
+  const RESPONSE_BODY = {
+    name: somethingLike('caseflag example'),
+    hearingRelevant: somethingLike(true),
+    flagComment: somethingLike(false),
+    flagCode: somethingLike('xflag'),
+    isParent: somethingLike(false),
+    Path: somethingLike(['/flag1']),
+    childFlags: somethingLike([]),
+    defaultStatus: somethingLike('Active'),
+    externallyActive: somethingLike(false)
```

Comment:
@connorpgpmcelroy - property name should be `externallyAvailable` as per comment below